specification/dimensions/skupriceid.md

```diff
@@ -0,0 +1,36 @@
+### SKU Price ID
+
+A SKU Price ID is a unique identifier that defines the price that is used to bill the SKU of the charge. This identifier represents a unique collection of nullable attributes that determine the price. SKU Price ID can be referenced on a price list published by a provider to look up detailed information about the SKU and the unit price associated with the SKU. The composition of the detailed information associated with the price may differ from one provider to the next.
+
+Every SKU has one or more SKU Prices.  For providers that define their prices at the SKU level, the SKU ID and SKU Price ID shall be the same.
```

Comment:
Wondering if others think this should be incorporated into the middle of the above paragraph. Typically, we try to avoid a second description paragraph for a column unless there's a lot to describe (there are cases where we describe a concept like 'charge period' in the first paragraph because we didn't have the glossary in 0.5 but those will be removed and moved to the glossary in v1.0.

Reply:
Feel free to make a suggestion and I'll incorporate.